### ISSUE TITLE
sirepo.run no attr handle

### ISSUE THREAD

**Issue by e-carlin** [radiasoft/sirepo#5473]
`curl sirepo.run | bash -s` then visit the home page

```
[2023-02-10 22:04:47,461] ERROR in app: Exception on /en/css/landing.css [GET]
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/uri_router.py", line 356, in _flask_dispatch
    reply_op=lambda r: r.flask_response(_app.response_class),
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/uri_router.py", line 319, in _call_api
    return reply_op(r) if reply_op else r
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/uri_router.py", line 356, in <lambda>
    reply_op=lambda r: r.flask_response(_app.response_class),
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/reply.py", line 151, in flask_response
    r = _file()
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/reply.py", line 137, in _file
    c.handle,
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/pykern/pkcollections.py", line 69, in __getattr__
    f"'{type(self).__name__}' object has no attribute '{name}'"
AttributeError: 'PKDict' object has no attribute 'handle'
```

**Comment by robnagler** [comment]
Verify this happens on dev as well. I feel like I fixed this.

**Comment by e-carlin** [comment]
Happening on dev as well.

**Comment by robnagler** [comment]
Awaiting review by @rorour on https://github.com/radiasoft/sirepo/pull/5452. You can review to speed things up.

**Comment by robnagler** [comment]
Fix #5452 which will automerge.
